### ISSUE TITLE
Jaws of Life cannot open windoors

### ISSUE THREAD

**Issue by williamcll** [BeeStation/NSV13#2813]
## Reproduction:
>Obtain Jaws of life
>Powered windoors resist from opening as if being opened by crowbar.
>No issues with powered Airlocks.

Mind you this was already fixed upstream on 
https://github.com/BeeStation/BeeStation-Hornet/pull/8620 and subsequently https://github.com/BeeStation/BeeStation-Hornet/pull/9412 as well as other codebases not having this issue either.
